src/domain/configure_actuators.py

```diff
@@ -72,8 +74,33 @@ def __init__(self):
 
     def initialize_dialog(self):
         """Method to initialize dialog with existing values (if any)."""
-        # TODO: fill up logic
-        pass
+
+        if FastAPIActuatorServer.actuator_server:
+            self.ui.lineEdit_server_ip.setText(FastAPIActuatorServer.actuator_server.ip)
+            self.ui.lineEdit_server_port.setText(FastAPIActuatorServer.actuator_server.port)
+            self.ui.label_key_file.setText(FastAPIActuatorServer.actuator_server.key)
+            self.ui.label_cert_file.setText(FastAPIActuatorServer.actuator_server.certificate)
+        self.list_actuators_in_tab()
+
+    def list_actuators_in_tab(self):
+        """Method to list cameras in FTPCameras tab."""
+
+        i = 1
+        for key in Actuator.actuators:
+            if i > 1:
+                self.add_actuator()
+            actuator_id_ln = self.findChild(QLineEdit, f"lineEdit_actuator_id_{i}")
+            actuator_id_ln.setText(Actuator.actuators[key].id)
+            actuator_type_cb: QComboBox
+            actuator_type_cb = self.findChild(QComboBox, f"comboBox_actuator_type_{i}")
+            if Actuator.actuators[key].type == Actuator.ActuatorTypes.ROADSIGN:
+                actuator_type_cb.setCurrentText(Actuator.ActuatorTypes.ROADSIGN.value)
+            elif Actuator.actuators[key].type == Actuator.ActuatorTypes.FEEDER:
+                actuator_type_cb.setCurrentText(Actuator.ActuatorTypes.FEEDER.value)
+            actuator_enablement: QCheckBox
```

Comment:
Redundant?

Reply:
I see it also on other places. Any reason for using type annotations only on a few places?

Reply:
not sure what you mean. it's also in mainwindow. but consider that not all logic about actuators is implemented yet.

Reply:
I mean that usually code is using type annotation (like in that line) or not.
To be more clear, there are people/projects that prefer to enforce a strong typing with the annotations, and other that keep the usual/original Python way without type annotations.
The second one is used by WADAS, but from time to time I see very sporadic occurrence of type annotations. Which is unusual. Hence my question: was there a specific reason for introducing it only there (and in a few other places like that)?

Reply:
Oh I got that. It's just useful when trying to access methods that otherwise are not listed by the IDE. We can remove it once we go towards stable release...